test/blackbox-tests/test-cases/menhir/cmly/run.t

```diff
@@ -1,14 +1,3 @@
-  $ dune build test.exe --display short --debug-dependency-path
-      ocamllex lexer1.ml
-      ocamldep .test.eobjs/test.ml.d
-        menhir test_menhir1.{cmly,ml,mli}
-      ocamldep .test.eobjs/lexer1.ml.d
-      ocamldep .test.eobjs/test_menhir1.mli.d
-      ocamldep .test.eobjs/test_menhir1.ml.d
-        ocamlc .test.eobjs/byte/test_menhir1.{cmi,cmti}
-      ocamlopt .test.eobjs/native/test_menhir1.{cmx,o}
-        ocamlc .test.eobjs/byte/lexer1.{cmi,cmo,cmt}
-        ocamlc .test.eobjs/byte/test.{cmi,cmo,cmt}
-      ocamlopt .test.eobjs/native/lexer1.{cmx,o}
-      ocamlopt .test.eobjs/native/test.{cmx,o}
-      ocamlopt test.exe
+  $ dune exec ./test.exe --debug-dependency-path
+  Fatal error: exception Failure("lexing: empty token")
+  [2]
```

Comment:
Not a huge deal that this test is failing given that we just want to test that `test.exe` is building, but it is a tad distracting. What about building and then using `ls` to check the exe is built? Or passing a valid input to the binary and make it print something.